src/Components/Notifications/NotificationPartnerShow.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from "react"
+import { graphql, useFragment } from "react-relay"
+import { RouterLink, RouterLinkProps } from "System/Router/RouterLink"
+import { AuthContextModule } from "@artsy/cohesion"
+import { Box, Button, Image, ResponsiveBox, Spacer, Text } from "@artsy/palette"
+import { NotificationPartnerShow_show$key } from "__generated__/NotificationPartnerShow_show.graphql"
+import { compact, truncate } from "lodash"
+import { CellShowStatus } from "Components/Cells/CellShow"
+
+const MAX_WIDTH = 600
```

Comment:
**suggestion / non-blocking**: there is a constant for max width [here](https://github.com/artsy/force/blob/main/src/Components/Notifications/constants.ts), let's use it?